src/api/subscriptions.ts

```diff
@@ -512,6 +512,11 @@ export interface OffersRequest {
    * upgrade or downgrade their current subscription.
    */
   oldSku?: string;
+
+  /**
+   * Optional. Sets if the fade in of is animated.
+   */
+  shouldNotAnimateFade?: boolean;
```

Comment:
can we make them all `shouldAnimateFade` for consistency?

Reply:
Done.

Reply:
I set them all to `shouldNotAnimateFade` so that if the value is unset, it will default to animating to preserve current behavior.

Reply:
I agree, the logic is hard to follow with the double negations.

Also consider that this is updating the public API. Is this necessary? Can we pass this option in another way?

If we do need to update the API, it's important that we keep it readable. IMO it would be better to name this positively with a comment that it defaults to true, and implement accordingly.

Reply:
If the current implementation requires updating the API, can you add a TODO to revisit? This UI detail doesn't really belong in OffersRequest.

Reply:
Done. Updated the comment and filed a bug to clean this up later.